Content.Server/Weapons/Melee/InfectOnMelee/InfectOnMeleeSystem.cs

```diff
@@ -0,0 +1,43 @@
+using Robust.Shared.Random;
+using Content.Shared.Cluwne;
+using Content.Shared.Interaction.Components;
+using Content.Shared.Humanoid;
+using Content.Shared.Mobs.Systems;
+using Content.Shared.Weapons.Melee.Events;
+using Content.Shared.Zombies;
+using Content.Shared.Weapons.Melee;
+
+namespace Content.Server.Weapons.Melee.InfectOnMelee;
+
+public sealed class InfectOnMeleeSystem : EntitySystem
+{
+    [Dependency] private readonly IRobustRandom _random = default!;
+    [Dependency] private readonly MobStateSystem _mob = default!;
+    [Dependency] private readonly SharedAudioSystem _audio = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<InfectOnMeleeComponent, MeleeHitEvent>(OnMeleeHit);
+    }
+
+    private void OnMeleeHit(EntityUid uid, InfectOnMeleeComponent component, MeleeHitEvent args)
+    {
+        if (component.Cluwinification == true)
+        {
```

Comment:
Please invert the condition to remove one level of indentation and you also don't need to check if it's true as per C# style-guide.